packages/build-rpm-packages.sh

```diff
@@ -101,7 +101,7 @@ FULL_VERSION="${VERSION}"
 
 if [[ ${generate_kmod} == 1 ]]; then
     if [ -z "${NVIDIA_SRC_DIR}" ]; then
-        NVIDIA_SRC_DIR=$(find /usr/src/nvidia-* -name "nv-p2p.h" -print -quit)
+        NVIDIA_SRC_DIR=$(find $(find /usr/src/ -type d -a -name nvidia-* -prune) -name "nv-p2p.h" -print -quit)
```

Comment:
If `$(find /usr/src/ -type d -a -name nvidia-* -prune)` outputs blank (cannot find any nvidia source code), the outer `find` will search the current directory and its sub directories. It could have unintended result if the current directory or its sub directory contains nv-p2p.h.

Reply:
Ok, I've looked into this some more, it seems there are only two possible location of the nvidia-* directory:
/usr/src and /usr/src/kernel-modules. So the test can be much more specific.